autoload/ale/fixers/standard.vim

```diff
@@ -0,0 +1,19 @@
+" Author: Sumner Evans <sumner.evans98@gmail.com>
+" Description: Fixing files with Standard.
+
+function! ale#fixers#standard#GetExecutable(buffer) abort
+    return ale#node#FindExecutable(a:buffer, 'javascript_standard', [
+    \   'node_modules/standard/bin/cmd.js',
```

Comment:
You should probably ensure that the `cmd.js` version is executed with `node` in the `Fix` function below on Windows. See the eslint function for an example.